src/Modal/index.jsx

```diff
@@ -43,6 +58,10 @@ class Modal extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    modalRoot.removeChild(this.el);
```

Comment:
Just out of curiosity are we missing anything that we'd get from [`ReactDOM.unmountComponentAtNode`](https://reactjs.org/docs/react-dom.html#unmountcomponentatnode)?

Reply:
I'm not sure, the docs for unmountComponentAtNode specifically mentions cleaning up eventListeners and such, which removeChild does not. I'll check to see if unmountComponentAtNode accomplishes what we want.